provider/gameround_controller.go

```diff
@@ -22,7 +22,8 @@ func (ctrl *GameRoundController) GetGameRoundEndpoint(c *fiber.Ctx) error {
 	if err != nil {
 		herr := &rest.HTTPError{}
 		if errors.As(err, herr) {
-			return c.Status(herr.Code).SendString(herr.Error())
+			c.Set(fiber.HeaderContentType, fiber.MIMETextHTML)
+			return c.Status(herr.Code).SendString(herr.Message)
```

Comment:
Does the message contain HTML? 😃

Reply:
yes, in some cases. Others just plain string, but that works with html content type as well